rllib/algorithms/impala/impala_learner.py

```diff
@@ -76,13 +80,17 @@ def additional_update_per_module(
     @override(Learner)
     def compile_results(
         self,
+        *,
         batch: MultiAgentBatch,
         fwd_out: Mapping[str, Any],
-        postprocessed_loss: Mapping[str, Any],
-        postprocessed_gradients: Mapping[str, Any],
+        loss_per_module: Union[TensorType, Mapping[str, Any]],
+        postprocessed_gradients: ParamDictType,
     ) -> Mapping[str, Any]:
         results = super().compile_results(
-            batch, fwd_out, postprocessed_loss, postprocessed_gradients
+            batch=batch,
```

Comment:
made these mandatory named args for clarity. Also renamed loss into `loss_per_module` (b/c it's a dict with all the modules' losses in it, including the `ALL_MODULES` total loss.